raw_tiles/source/osm.py

```diff
@@ -3,6 +3,13 @@
 from raw_tiles.util import time_block
 
 
+def convert_rels_null_tags_to_empty_list(row):
+    assert len(row) == 6, 'Unexpected number of columns: %d' % len(row)
+    if row[5] is None:
+        row[5] = []
+    return row
```

Comment:
@zerebubuth it might be better and more efficient to do this at the query via something like coalesce, but this works for now.

Reply:
Agreed. Although part of me wonders whether the array is null in the database, or that it somehow gets converted during serialisation/deserialisation when being transferred back to the client.